source/aks/upgrading-aks-clusters.html.md.erb

```diff
@@ -25,7 +25,9 @@ This is the normal order we use when upgrading clusters, it does not have to exa
 
 ## Pods and HRs Health Checks
 
-Put a message out into the #cloud-native-announce channel to notify people which cluster will be upgraded (do not use @here as this is a notification not an announcement).
+Please post a notification in the #cloud-native-announce channel to notify people which clusters will be upgraded (Please refrain from using @here, as this is intended for notifications rather than announcements).
+Utilize the #sds-cloud-native channel if you are involved in activities concerning SDS clusters. 
```

Comment:
is all this needed? we want to keep our documentation short.

Reply:
Yes because we are only providing information on the CFT cluster and we need to ensure that if a user is looking at this they are aware of the steps to take if they are looking at SDS clusters.